src/core/import/qImport.ts

```diff
@@ -35,9 +33,9 @@ export function qImport<T>(node: Node | Document, url: string | QRL<T> | URL): T
       return Promise.resolve<T>(testSymbol);
     }
   }
-  const doc: QDocument = node.ownerDocument || (node as Document);
+  const doc: QDocument = element.ownerDocument!;
```

Comment:
Not possible for `document` to be an element?

Reply:
not any more.